tests/app/main/views/test_services.py

```diff
@@ -62,65 +75,88 @@ class TestServiceView(LoggedInApplicationTest):
         },
     ]}
 
-    # def test_service_view_status_disabled(self, data_api_client):
-    #     data_api_client.get_service.return_value = {'services': {
-    #         'frameworkSlug': 'g-cloud-8',
-    #         'serviceName': 'test',
-    #         'supplierId': 1000,
-    #         'lot': 'iaas',
-    #         'id': "314159265",
-    #         "status": "disabled",
-    #     }}
-    #     data_api_client.find_audit_events.return_value = self.find_audit_events_api_response
-    #     response = self.client.get('/admin/services/314159265')
-    #
-    #     assert response.status_code == 200
-    #     assert data_api_client.get_service.call_args_list == [
-    #         (("314159265",), {}),
-    #     ]
-    #     assert data_api_client.find_audit_events.call_args_list == [
-    #         mock.call(
-    #             audit_type=AuditTypes.update_service_status,
-    #             latest_first='true',
-    #             object_id='314159265',
-    #             object_type='services'
-    #         )
-    #     ]
-    #
-    #     document = html.fromstring(response.get_data(as_text=True))
-    #     assert document.xpath(
-    #         "normalize-space(string(//td[@class='summary-item-field']//*[@class='service-id']))"
-    #     ) == "314159265"
-    #
-    # def test_service_view_status_enabled(self, data_api_client):
-    #     data_api_client.get_service.return_value = {'services': {
-    #         'frameworkSlug': 'g-cloud-7',
-    #         'serviceName': 'test',
-    #         'supplierId': 1000,
-    #         'lot': 'iaas',
-    #         'id': "1412",
-    #         "status": "enabled",
-    #     }}
-    #     data_api_client.find_audit_events.return_value = self.find_audit_events_api_response
-    #     response = self.client.get('/admin/services/1412')
-    #
-    #     assert response.status_code == 200
-    #     assert data_api_client.get_service.call_args_list == [
-    #         (("1412",), {}),
-    #     ]
-    #     assert data_api_client.find_audit_events.call_args_list == [
-    #         mock.call(
-    #             audit_type=AuditTypes.update_service_status,
-    #             latest_first='true',
-    #             object_id='1412',
-    #             object_type='services'
-    #         )
-    #     ]
-    #
-    #     document = html.fromstring(response.get_data(as_text=True))
-    #     assert document.xpath(
-    #         "normalize-space(string(//td[@class='summary-item-field']//*[@class='service-id']))"
-    #     ) == "1412"
+    def test_service_view_status_disabled(self, data_api_client):
```

Comment:
I think these tests should have been deleted, rather than reinstated? That three-radio-button form that these are testing is old news.

Reply:
Rebase problems again 😿